mappings/net/minecraft/block/entity/PistonBlockEntity.mapping

```diff
@@ -57,7 +57,9 @@ CLASS net/minecraft/class_2669 net/minecraft/block/entity/PistonBlockEntity
 		ARG 0 box
 		ARG 1 entity
 	METHOD method_23672 moveEntity (Lnet/minecraft/class_2350;Lnet/minecraft/class_1297;DLnet/minecraft/class_2350;)V
+		ARG 0 direction
 		ARG 1 entity
+		ARG 4 oppositeDirection
```

Comment:
I don't think this is accurate. Two usages pass the same direction.

Reply:
now better named